Formula/enchant.rb

```diff
@@ -24,11 +24,15 @@ def install
                           "--enable-relocatable"
 
     system "make", "install"
+    if OS.mac?
+      # Use AppleSpell for all languages by default
+      system "echo *:AppleSpell > #{share}/enchant/enchant.ordering"
+    end
     ln_s "enchant-2.pc", lib/"pkgconfig/enchant.pc"
   end
 
   test do
-    text = "Teh quikc brwon fox iumpz ovr teh lAzy d0g"
+    text = "Teh quikc brwon fox iumpz ovr teh"
```

Comment:
Why change this?

Reply:
The AppleSpell spell checker did not detect lAzy and d0g as misspelled, making the test fail.

Reply:
I'd say that's an argument against this change

Reply:
I don't agree.  The test does not test the correct functioning of enchant but only whether the default underlying spell checker (aspell in the homebrew version) finds a few specific typos.  Thus, it is specific for aspell and English.  The AppleSpell checker provides automatic language detection and is the default for most apps on macs.